app/lib/wizard_state_stores/redis_store.rb

```diff
@@ -0,0 +1,22 @@
+require 'redis'
+
+module WizardStateStores
+  class RedisStore
+    def initialize(key:)
+      @redis = Redis.new
```

Comment:
```suggestion
      @redis = Redis.current
```

works well for re-using connections.